gfx/models/portraits/attachments/male_headgears/secular/goblin/gallywix_hat_01/GallywixHat.asset

```diff
@@ -0,0 +1,19 @@
+pdxmesh = {
+	name = "GallywixHat_mesh"
+	file = "GallywixHat.mesh"
+
+	meshsettings = {
+		name = "GallywixHatShape"
+		index = 0
+		texture_diffuse = "GallywixHatDiffuse.png"
+		texture_normal = "GallywixHatNormal.png"
+		texture_specular = "GallywixHatProperties.png"
+		shader = "standard"
```

Comment:
Shader with such a name doesn't exist. Paradox use `portrait_attachment` shader for headgears.